utility/namespace.py

```diff
@@ -25,5 +25,5 @@ def create_private_namespace(name, region='us-east-2'):
     if not os.path.exists(local_dir):
         os.makedirs(local_dir)
     templ_parm = templating.Templating()
-    create_oc_resource('private.yaml', local_dir, templ_parm,
+    create_oc_resource('namespace.yaml', local_dir, templ_parm,
```

Comment:
fix parenthesis according to the coding guidelines

Reply:
Done